lib/node_modules/@stdlib/blas/base/drotg/test/test.assign.js

```diff
@@ -0,0 +1,243 @@
+	t.strictEqual( drotg.assign.length, 5, 'arity of 5' );
+	t.end();
+});
+
+tape( 'the function computes a Givens plane rotation', function test( t ) {
+	var expected;
+	var values;
+	var delta;
+	var tol;
+	var out;
+	var e;
+	var i;
+	var j;
+
+	expected = [
+		[ 0.5, 1.0 / 0.6, 0.6, 0.8 ],
+		[ 0.5, 0.6, 0.8, 0.6 ],
+		[ 0.5, -1.0 / 0.6, -0.6, 0.8 ],
+		[ -0.5, -0.6, 0.8, -0.6 ],
+		[ -0.5, 1.0 / 0.6, 0.6, 0.8 ],
+		[ 0.0, 0.0, 1.0, 0.0 ],
+		[ 1.0, 1.0, 0.0, 1.0 ],
+		[ 1.0, 0.0, 1.0, 0.0 ]
+	];
+	values = [
+		[ 0.3, 0.4 ],
+		[ 0.4, 0.3 ],
+		[ -0.3, 0.4 ],
+		[ -0.4, 0.3 ],
+		[ -0.3, -0.4 ],
+		[ 0.0, 0.0 ],
+		[ 0.0, 1.0 ],
+		[ 1.0, 0.0 ]
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		e = new Float64Array( expected[i] );
+		out = new Float64Array( 4 );
+		drotg.assign( values[i][0], values[i][1], out, 1, 0 );
+		for ( j = 0; j < out.length; j++ ) {
+			if ( out[j] === e[j] ) {
+				t.strictEqual( out[j], e[j], 'returns expected value' );
+			} else {
+				delta = abs( out[j] - e[j] );
+				tol = 1.5 * EPS * abs( e[j] );
+				t.ok( delta <= tol, 'within tolerance. out: '+out[j]+'. expected: '+e[j]+'. delta: '+delta+'. tol: '+tol+'.' );
+			}
+		}
+	}
+	t.end();
+});
+
+tape( 'the function returns an array of NaNs if provided a rotation elimination parameter equal to NaN', function test(t) {
+	var actual;
+	var i;
+
+	actual = drotg( NaN, 1.0, new Float64Array( 4 ), 1, 0 );
+	for ( i = 0; i < actual.length; i++ ) {
+		t.strictEqual( isnan( actual[i] ), true, 'returns expected value' );
+	}
+
+	actual = drotg( 1.0, NaN, new Float64Array( 4 ), 1, 0 );
+	for ( i = 0; i < actual.length; i++ ) {
+		t.strictEqual( isnan( actual[i] ), true, 'returns expected value' );
+	}
+
+	actual = drotg( NaN, NaN, new Float64Array( 4 ), 1, 0 );
+	for ( i = 0; i < actual.length; i++ ) {
+		t.strictEqual( isnan( actual[i] ), true, 'returns expected value' );
+	}
+	t.end();
+});
+
+tape( 'the function supports providing a positive stride', function test(t) {
+	var expected;
+	var actual;
+	var delta;
+	var tol;
+	var out;
+	var i;
+
+	expected = new Float64Array( [ 0.5, 0.0, 1.0 / 0.6, 0.0, 0.6, 0.0, 0.8 ] );
+	out = new Float64Array( [ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 ] );
+
+	actual = drotg.assign( 0.3, 0.4, out, 2, 0 );
+	for ( i = 0; i < out.length; i++ ) {
+		if ( out[i] === expected[i] ) {
+			t.strictEqual( out[i], expected[i], 'returns expected value' );
+		} else {
+			delta = abs( out[i] - expected[i] );
+			tol = 1.5 * EPS * abs( expected[i] );
+			t.ok( delta <= tol, 'within tolerance. out: '+out[i]+'. expected: '+expected[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
+	for ( i = 0; i < out.length; i++ ) {
+		if ( out[i] === actual[i] ) {
+			t.strictEqual( out[i], actual[i], 'returns expected value' );
+		} else {
+			delta = abs( out[i] - actual[i] );
+			tol = 1.5 * EPS * abs( actual[i] );
+			t.ok( delta <= tol, 'within tolerance. out: '+out[i]+'. expected: '+actual[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
```

Comment:
```suggestion
```

This appears to be duplicated.

Reply:
Ah. Okay, I see what you are doing here. This is unnecessary. We just need to test whether `actual === out`.